pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -268,9 +321,27 @@ let
       # weaken or disable security measures like sandboxing or ASLR):
       is_official_build = true;
       disable_fieldtrial_testing_config = true;
+
+      # note: chromium calls buildPlatform "host" and calls hostPlatform "target"
+      host_cpu      = chromiumRosettaStone.cpu stdenv.buildPlatform;
+      host_os       = chromiumRosettaStone.os  stdenv.buildPlatform;
+      target_cpu    = chromiumRosettaStone.cpu stdenv.hostPlatform;
+      v8_target_cpu = chromiumRosettaStone.cpu stdenv.hostPlatform;
+      target_os     = chromiumRosettaStone.os  stdenv.hostPlatform;
```

Comment:
Setting targets may be the cause of always cross-compiling

https://github.com/void-linux/void-packages/blob/7b23d90a5ea77b8d91f12c818d620e90eb710b3c/srcpkgs/chromium/template#L250-L281

Reply:
Possible improvement:

- https://github.com/NixOS/nixpkgs/pull/264326